### PR TITLE
update scripts to work on non-windows and match some Collider & LiveActor functions

### PR DESCRIPTION
**Pull request by LDAsuku**
scripts have been updated
- it works on linux
- it verifies hash of fury.nso

matched some functions
- some trivial LiveActor functions as well as Collider constructor and fields

### CODE REVIEW

File: lib/ActionLibrary/source/Library/Collision/Collider.cpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+namespace al {
+    Collider::Collider(CollisionDirector *pCollisionDirector, const sead::Matrix34f *pBaseMtx, const sead::Vector3f *pTrans, const sead::Vector3f *pGravity, f32 radius, f32 offsetY, u32 numHitInfos) :
+            mCollisionDirector(pCollisionDirector),
+            mBaseMtx(pBaseMtx),
+            mTrans(pTrans),
+            mGravity(pGravity),
+            mRadius(radius),
+            mOffsetY(offsetY),
+            _48(numHitInfos),
+            _58(sead::Vector3f(0.0f, 0.0f, 0.0f)),
```
Comment (shibbo) [inline lib/ActionLibrary/source/Library/Collision/Collider.cpp:13]: These are most likely implicitly constructed to 0. Can we see if removing these still matches?

**Review by shibbo — COMMENTED** [review]
Just one tiny detail to look at before we can push

File: lib/ActionLibrary/source/Library/Collision/Collider.cpp (hunk 1 above)
Comment (LDAsuku) [reply lib/ActionLibrary/source/Library/Collision/Collider.cpp:13]: assuming you're referring to _58 and _64: nope, they do get stored to twice. removing them causes it to not match

Comment (shibbo) [reply lib/ActionLibrary/source/Library/Collision/Collider.cpp:13]: Sounds good then.

**Pull request merged**